client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/screen-homepage.tsx

```diff
@@ -0,0 +1,67 @@
+import { Button } from '@automattic/components';
+import {
+	__experimentalNavigatorBackButton as NavigatorBackButton,
+	__experimentalUseNavigator as useNavigator,
+} from '@wordpress/components';
+import { useTranslate } from 'i18n-calypso';
+import NavigatorHeader from './navigator-header';
+import PatternLayout from './pattern-layout';
+import type { Pattern } from './types';
+
+interface Props {
+	patterns: Pattern[];
+	onDoneClick: () => void;
+	onReplaceSection: ( position: number ) => void;
+	onDeleteSection: ( position: number ) => void;
+	onMoveUpSection: ( position: number ) => void;
+	onMoveDownSection: ( position: number ) => void;
+}
+
+const ScreenHomepage = ( {
```

Comment:
The new layout of "Design your home"

<img width="195" alt="image" src="https://user-images.githubusercontent.com/13596067/220596201-841faf09-fd76-4533-8d13-a40155f66962.png">